### DISCUSSION TITLE
QMK customization

### DISCUSSION THREAD

**Discussion by AuditeMarlow** [manna-harbour/miryoku#240] [Q&A] [answered]
How do I customize, for example, the default layers of the Miryoku layout? Additionally, I'd like to be able to do things like retro tapping, tap dance, etc. Is there some documentation available for this?

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/discussions/85.

**Comment by AuditeMarlow** [comment]
Thanks for providing me with an answer, @manna-harbour. Additionally I was wondering if I can enable some of the now-dead keys on my keyboard? I have a split_3x6_3 keyboard and would like to use the outer most keys in a custom layout as well. Is this possible?

**Reply by manna-harbour** [reply]
Yes, to map extra keys see the _Mapping_ section of the document linked above.

**Reply by AuditeMarlow** [reply]
Just what I needed, thanks!
